biztown/src/components/FormInputLayout.js

```diff
@@ -15,30 +15,48 @@ class FormInputLayout extends Component {
         this.state = {
             currentStep: 1,
             aboutData: {},
+            schoolData: {},
             multipleChoiceData: {},
-            password: '', 
-            pageTitle: [
-                'Tell Us About You',
-                'Questions About The Program Content',
-                'Fill in the appropriate blanks to complete the checkbook components'
-            ]
+            freeResponseData: {
```

Comment:
add prepopulated data for free response instead of manually adding individual rows